content/docs/tasks/security/authz-tcp/index.md

```diff
@@ -0,0 +1,231 @@
+    serviceaccount "bookinfo-ratings-v2" created
+    deployment "ratings-v2" configured
+    {{< /text >}}
+
+    If you are using manual sidecar injection, use the following command instead:
+
+    {{< text bash >}}
+    $ kubectl apply -f <(istioctl kube-inject -f @samples/bookinfo/platform/kube/rbac/ratings-v2-add-serviceaccount.yaml@)
+    serviceaccount "bookinfo-ratings-v2" created
+    deployment "ratings-v2" configured
+    {{< /text >}}
+
+1. Update the Bookinfo sample to use the new version of `ratings`:
+
+    The Bookinfo sample deploys multiple versions of each microservice, so you will start by creating
+    destination rules that define the service subsets corresponding to each version, and the load
+    balancing policy for each subset.
+
+    {{< text bash >}}
+    $ kubectl apply -f @samples/bookinfo/networking/destination-rule-all-mtls.yaml@
+    destinationrule "productpage" created
+    destinationrule "reviews" created
+    destinationrule "ratings" created
+    destinationrule "details" created
+    {{< /text >}}
+
+    Since the subset referenced in virtual services rely on the destination rules,
+    wait a few seconds for destination rules to propagate before adding virtual services that refer
+    to these subsets.
+
+    Update the `reviews` service to only use the `v2` of `ratings` service with the following command:
+
+    {{< text bash >}}
+    $ kubectl apply -f @samples/bookinfo/networking/virtual-service-ratings-db.yaml@
+    virtualservice "reviews" created
+    virtualservice "ratings" created
+    {{< /text >}}
+
+    Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`). You should see:
+
+       * The "Book Details" section in the lower left part of the page, including type, pages, publisher, etc.
+       * The "Book Reviews" section in the lower right part of the page with "Ratings service is currently unavailable",
+         This is because we switched to use `v2` of `ratings` but haven't deployed the mongoDB service.
+
+1. Deploy the mongoDB service:
+
+    If you are using a cluster with automatic sidecar injection enabled,
+    simply deploy the services using `kubectl`:
+
+    {{< text bash >}}
+    $ kubectl apply -f @samples/bookinfo/platform/kube/bookinfo-db.yaml@
+    {{< /text >}}
+
+    If you are using manual sidecar injection, use the following command instead:
+
+    {{< text bash >}}
+    $ kubectl apply -f <(istioctl kube-inject -f @samples/bookinfo/platform/kube/bookinfo-db.yaml@)
+    service "mongodb" configured
+    deployment "mongodb-v1" configured
+    {{< /text >}}
+
+    Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`). You should see:
+
+       * The "Book Details" section in the lower left part of the page, including type, pages, publisher, etc.
+       * The "Book Reviews" section in the lower right part of the page with red stars.
```

Comment:
Enhance brevity and avoid needless repetition. Lines 118-121 can be summarized in numbered steps:
1. Go to the Bookinfo product page at (http://$GATEWAY_URL/productpage)
1. Verify that the \*\*Book Reviews\*\* section shows the reviews.